README.md

```diff
@@ -133,6 +133,7 @@
 * [namaka](https://github.com/nix-community/namaka) - Snapshot testing for Nix based on haumea.
 * [nil](https://github.com/oxalica/nil) - NIx Language server, an incremental analysis assistent for writing in Nix.
 * [niv](https://github.com/nmattia/niv/) - Easy dependency management for Nix projects with package pinning.
+* [nixd](https://github.com/nix-community/nixd) - Nix language server, based on nix libraries.
```

Comment:
Capitalize "nix" please, and move the entry below "nix-update" to maintain alphabetical sorting (I admittedly just use (neo)vim's sort function).

Reply:
done